tests/test_instationary_rotational_couette.py

```diff
@@ -0,0 +1,78 @@
+import dolfin as dlfn
+from ns_problem import InstationaryProblem
+from ns_bdf_solver import ImplicitBDFSolver
+from ns_solver_base import VelocityBCType
+from ns_solver_base import PressureBCType
+from ns_solver_base import TractionBCType
+from grid_generator import hyper_cube
+from grid_generator import hyper_rectangle
+from grid_generator import open_hyper_cube
+from grid_generator import HyperCubeBoundaryMarkers
+from grid_generator import HyperRectangleBoundaryMarkers
+from grid_generator import spherical_shell
+from grid_generator import SphericalAnnulusBoundaryMarkers
+
+dlfn.set_log_level(20)
+
+
+class InstatRotationalCouetteFlow(InstationaryProblem):
+    def __init__(self, n_points, n_refinements, radii, main_dir=None):
+        super().__init__(main_dir, start_time=0.0, end_time=20.0,
+                         desired_start_time_step=0.0025, n_max_steps=5)
+
+        # parameters for mesh
+        self._radii = radii
+        self._n_points = n_points
+        self._n_refinements = n_refinements
+        self._problem_name = "InstatRotationalCouette"
+
+        # dimensionless parameters and starting values of Omega and Alpha = dOmega/dt
+        # Alpha is considered to be constant until t_acceleration, after that it is 0
+        # Note that _acceleration_time has not yet been implemented in ns_bdf_solver; for now it is considered to be 1.0
+        omega_start = 0
+        self._alpha_start = 1
+        self._acceleration_time = 1.0
+        self.set_parameters(Re=550.0, Ro=1.0, Omega=dlfn.Constant(omega_start), Alpha=dlfn.Constant(self._alpha_start))
```

Comment:
This test failes because the method `set_parameters` does not exist anymore. I have modified the code structure and the handling of dimensionless parameters recently. Please have a look at the other tests how the parameters should be specified now.